applications/GeoMechanicsApplication/tests/cpp_tests/test_geomechanics_erosion_piping.cpp

```diff
@@ -0,0 +1,215 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include <string>
+
+// Project includes
+#include "containers/model.h"
+#include "custom_constitutive/linear_elastic_2D_interface_law.h"
+#include "custom_elements/U_Pw_small_strain_element.hpp"
+#include "custom_elements/plane_strain_stress_state.h"
+#include "custom_elements/steady_state_Pw_piping_element.hpp"
+#include "custom_strategies/strategies/geo_mechanics_newton_raphson_erosion_process_strategy.hpp"
+#include "custom_workflows/dgeoflow.h"
+#include "geo_mechanics_fast_suite.h"
+#include "test_utilities.h"
+
+namespace Kratos
+{
+template <class TSparseSpace, class TDenseSpace, class TLinearSolver>
+class MockGeoMechanicsNewtonRaphsonErosionProcessStrategy
+    : public GeoMechanicsNewtonRaphsonErosionProcessStrategy<TSparseSpace, TDenseSpace, TLinearSolver>
+{
+public:
+    using BaseType = ImplicitSolvingStrategy<TSparseSpace, TDenseSpace, TLinearSolver>;
+    using TConvergenceCriteriaType = ConvergenceCriteria<TSparseSpace, TDenseSpace>;
+    using TBuilderAndSolverType    = typename BaseType::TBuilderAndSolverType;
+    using TSchemeType              = typename BaseType::TSchemeType;
+
+    MockGeoMechanicsNewtonRaphsonErosionProcessStrategy(ModelPart&                    rModelPart,
+                                                        typename TSchemeType::Pointer pScheme,
+                                                        typename TLinearSolver::Pointer pNewLinearSolver,
+                                                        typename TConvergenceCriteriaType::Pointer pNewConvergenceCriteria,
+                                                        typename TBuilderAndSolverType::Pointer pNewBuilderAndSolver,
+                                                        Parameters& rParameters)
+        : GeoMechanicsNewtonRaphsonErosionProcessStrategy<TSparseSpace, TDenseSpace, TLinearSolver>(
+              rModelPart, pScheme, pNewLinearSolver, pNewConvergenceCriteria, pNewBuilderAndSolver, rParameters),
+          mrModelPart(rModelPart) {};
```

Comment:
The semicolon is redundant here, so please remove it.
```suggestion
          mrModelPart(rModelPart) {}
```

Reply:
removed